[2024-12-17T06:50:00.756+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: full_name_dag.extract_data scheduled__2024-12-16T00:00:00+00:00 [queued]>
[2024-12-17T06:50:00.764+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: full_name_dag.extract_data scheduled__2024-12-16T00:00:00+00:00 [queued]>
[2024-12-17T06:50:00.764+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 2
[2024-12-17T06:50:00.778+0000] {taskinstance.py:1380} INFO - Executing <Task(PythonOperator): extract_data> on 2024-12-16 00:00:00+00:00
[2024-12-17T06:50:00.783+0000] {standard_task_runner.py:57} INFO - Started process 9922 to run task
[2024-12-17T06:50:00.785+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'full_name_dag', 'extract_data', 'scheduled__2024-12-16T00:00:00+00:00', '--job-id', '113', '--raw', '--subdir', 'DAGS_FOLDER/full_name_dag.py', '--cfg-path', '/tmp/tmp6sz4uzj5']
[2024-12-17T06:50:00.788+0000] {standard_task_runner.py:85} INFO - Job 113: Subtask extract_data
[2024-12-17T06:50:00.811+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-12-17T06:50:00.848+0000] {task_command.py:415} INFO - Running <TaskInstance: full_name_dag.extract_data scheduled__2024-12-16T00:00:00+00:00 [running]> on host cfbf61ae1f16
[2024-12-17T06:50:00.964+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='full_name_dag' AIRFLOW_CTX_TASK_ID='extract_data' AIRFLOW_CTX_EXECUTION_DATE='2024-12-16T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-12-16T00:00:00+00:00'
[2024-12-17T06:50:01.261+0000] {python.py:194} INFO - Done. Returned value was: None
[2024-12-17T06:50:01.272+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=full_name_dag, task_id=extract_data, execution_date=20241216T000000, start_date=20241217T065000, end_date=20241217T065001
[2024-12-17T06:50:01.319+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-12-17T06:50:01.343+0000] {taskinstance.py:2776} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-12-17T09:12:19.177+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: full_name_dag.extract_data scheduled__2024-12-16T00:00:00+00:00 [queued]>
[2024-12-17T09:12:19.254+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: full_name_dag.extract_data scheduled__2024-12-16T00:00:00+00:00 [queued]>
[2024-12-17T09:12:19.255+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 2
[2024-12-17T09:12:19.266+0000] {taskinstance.py:1380} INFO - Executing <Task(PythonOperator): extract_data> on 2024-12-16 00:00:00+00:00
[2024-12-17T09:12:19.271+0000] {standard_task_runner.py:57} INFO - Started process 12654 to run task
[2024-12-17T09:12:19.272+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'full_name_dag', 'extract_data', 'scheduled__2024-12-16T00:00:00+00:00', '--job-id', '127', '--raw', '--subdir', 'DAGS_FOLDER/full_name_dag.py', '--cfg-path', '/tmp/tmpzo28c5ok']
[2024-12-17T09:12:19.275+0000] {standard_task_runner.py:85} INFO - Job 127: Subtask extract_data
[2024-12-17T09:12:19.290+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-12-17T09:12:19.324+0000] {task_command.py:415} INFO - Running <TaskInstance: full_name_dag.extract_data scheduled__2024-12-16T00:00:00+00:00 [running]> on host cfbf61ae1f16
[2024-12-17T09:12:19.394+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='full_name_dag' AIRFLOW_CTX_TASK_ID='extract_data' AIRFLOW_CTX_EXECUTION_DATE='2024-12-16T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-12-16T00:00:00+00:00'
[2024-12-17T09:12:19.557+0000] {python.py:194} INFO - Done. Returned value was: None
[2024-12-17T09:12:19.568+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=full_name_dag, task_id=extract_data, execution_date=20241216T000000, start_date=20241217T091219, end_date=20241217T091219
[2024-12-17T09:12:19.607+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-12-17T09:12:19.624+0000] {taskinstance.py:2776} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-12-17T09:13:07.819+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: full_name_dag.extract_data scheduled__2024-12-16T00:00:00+00:00 [queued]>
[2024-12-17T09:13:07.916+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: full_name_dag.extract_data scheduled__2024-12-16T00:00:00+00:00 [queued]>
[2024-12-17T09:13:07.916+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 2
[2024-12-17T09:13:07.928+0000] {taskinstance.py:1380} INFO - Executing <Task(PythonOperator): extract_data> on 2024-12-16 00:00:00+00:00
[2024-12-17T09:13:07.933+0000] {standard_task_runner.py:57} INFO - Started process 12696 to run task
[2024-12-17T09:13:07.935+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'full_name_dag', 'extract_data', 'scheduled__2024-12-16T00:00:00+00:00', '--job-id', '133', '--raw', '--subdir', 'DAGS_FOLDER/full_name_dag.py', '--cfg-path', '/tmp/tmpg3e4dc70']
[2024-12-17T09:13:07.937+0000] {standard_task_runner.py:85} INFO - Job 133: Subtask extract_data
[2024-12-17T09:13:07.954+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-12-17T09:13:07.991+0000] {task_command.py:415} INFO - Running <TaskInstance: full_name_dag.extract_data scheduled__2024-12-16T00:00:00+00:00 [running]> on host cfbf61ae1f16
[2024-12-17T09:13:08.071+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='full_name_dag' AIRFLOW_CTX_TASK_ID='extract_data' AIRFLOW_CTX_EXECUTION_DATE='2024-12-16T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-12-16T00:00:00+00:00'
[2024-12-17T09:13:08.235+0000] {python.py:194} INFO - Done. Returned value was: None
[2024-12-17T09:13:08.246+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=full_name_dag, task_id=extract_data, execution_date=20241216T000000, start_date=20241217T091307, end_date=20241217T091308
[2024-12-17T09:13:08.269+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-12-17T09:13:08.290+0000] {taskinstance.py:2776} INFO - 1 downstream tasks scheduled from follow-on schedule check
